frontend/src/entities/dataset/dataset.tsx

```diff
@@ -39,7 +39,7 @@ export const Dataset = () => {
     !projectName ? DocTitle('Datasets') : DocTitle(projectName!.concat(' Datasets'));
 
     useEffect(() => {
-        dispatch(getDatasetsList(projectName));
+        dispatch(getDatasetsList({projectName}));
```

Comment:
we probably dont need this anymore since we are passing it in the table below

Reply:
Happy to remove this, I was just fixing a broken function call.